src/components/pages/Home/Home.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import React, { useMemo } from "react";
+
+import { useRouter } from "next/navigation";
+import { useTranslations } from "next-intl";
+
+import { ITopLatestExhibit } from "@/interfaces/ITopLatestExhibit";
+import { IImageGalleryImage } from "@/interfaces/IImageGalleryImage";
+
+import ImageGallery from "@/components/organisms/ImageGallery/ImageGallery";
+
+import styles from "./Home.module.scss";
+
+interface Props {
+  exhibits: ITopLatestExhibit[];
+}
+
+export default function Home(props: Props): React.ReactElement {
+  const router = useRouter();
+  const t = useTranslations();
+
+  const images: IImageGalleryImage[] = useMemo(
+    () =>
+      props.exhibits.map((exhibit) => {
+        const firstImageItem = exhibit.imagesCollection.items[0];
+
+        return {
+          url: firstImageItem.url,
+          id: firstImageItem.sys.id,
+        };
+      }),
+    [props.exhibits],
+  );
+
+  const onClickImage = (id: string): void => {
+    const exhibit = props.exhibits.find(
+      (topLatestExhibit) => topLatestExhibit.imagesCollection.items[0].sys.id === id,
+    );
+
+    if (!exhibit) {
+      return;
+    }
+
+    router.push(`exhibit/${exhibit.slug}`);
```

Comment:
Let's extract "exhibit" into global constant, pretty sure that this will be used a lot in future